Sources/KeyboardShortcuts/RecorderCocoa.swift

```diff
@@ -28,7 +28,15 @@ extension KeyboardShortcuts {
 	public final class RecorderCocoa: NSSearchField, NSSearchFieldDelegate {
 		private let minimumWidth: Double = 130
 		private var eventMonitor: LocalEventMonitor?
-		private let shortcutName: Name
+		public var shortcutName: Name {
```

Comment:
I think this should have a short doc comment pointing out that it can be dynamically changed. I don't think that's obvious.